liquidity/liquidity.go

```diff
@@ -501,7 +516,38 @@ func (m *Manager) easyAutoLoop(ctx context.Context) error {
 	m.refreshAutoloopBudget(ctx)
 
 	// Dispatch the best easy autoloop swap.
-	err := m.dispatchBestEasyAutoloopSwap(ctx)
+	err := m.dispatchBestEasyAutoloopSwap(
```

Comment:
I think instead of passing an assetID here we could leave the legacy function as is and then add a custom `dispatchBestAssetEasyAutoloopSwap` method, which will only be called in `easyAutoLoopAsset`.

IIUC the current changes to `dispatchBestEasyAutoloopSwap` to support the asset swap are distinct from the legacy flow, so should be easy to carve out. Wdyt?